shared/js/background/before-request.es6.js

```diff
@@ -97,6 +97,17 @@ function handleRequest (requestData) {
     // don't have a tab instance for this tabId or this is a new requestId.
     if (requestData.type === 'main_frame') {
         if (!thisTab || thisTab.requestId !== requestData.requestId) {
+            if(thisTab) {
+                // upon navigation, remove any custom action session rules
+                const customRulesIds = []
+                for (const ruleSet in thisTab.customActionRules) {
+                    customRulesIds.push(...thisTab.customActionRules[ruleSet])
+                }
+                if (customRulesIds) {
```

Comment:
I think an empty array is considered truthy in JavaScript. How about this?
```suggestion
                if (customRulesIds.length === 0) {
```

Reply:
Good catch! Fixed a few of these but missed this one